dbfit-java/core/src/main/java/dbfit/api/AbstractDbEnvironment.java

```diff
@@ -246,5 +248,14 @@ private static boolean isConnected(final Connection conn) throws SQLException {
         return (conn != null && !conn.isClosed());
     }
 
+    private Map<Class<?>, TypeSpecifier> typeSpecifiers = new HashMap<Class<?>, TypeSpecifier>();
+
+    public TypeSpecifier getTypeSpecifier(Class<?> targetClass) {
+    	return typeSpecifiers.get(targetClass);
+    }
```

Comment:
Remember in the Informix adapter that I'm reusing/overloading `TypeNormaliserFactory` to cast back to a more specific type (the opposite of normalising) before binding to the `PreparedStatement` and we thought a separate interface would be more appropriate.

It doesn't feel quite right to have this mechanism as a global mapping as it's only required to make specific types for specific adapters.